format/applebookmark/apple_bookmark.md

```diff
@@ -0,0 +1,41 @@
+## Apple bookmarkData format
```

Comment:
You can skip the header, it's added by make doc and the help system. It will use the description.

I usually have a `help_<format>.fqtest` file in testdata with just `$ fq -h <format>` to test it.

Reply:
Having some issues with this related to the long lines again. `make test` is producing output where the lines have been broken to a fixed width, whereas the output I put into the `fqtest` file has the long lines unbroken.

Reply:
> Any suggestions for a good workflow for when I need to regenerate `fqtest` output after decoder changes? It's a bit tedious to go through, delete the old stuff, and rerun them individually; I suspect you might have a better way to do that.

Oh you copy/paste it manually, that should not be needed. I usually run something like this to update the expected output:
```
WRITE_ACTUAL=1 go test -v -run TestFormats/applebookmark ./format
```
Then it will rewrite the .fqtest files with the current output, you can verify after by running without `WRITE_ACTUAL`.

So my workflow for tests is usually that is create a fqtest file with just `$ fq ...` lines and then run the tests with `WRITE_ACTUAL` to get the output.

Sometimes when i'm quite sure a change needs an update is probably correct i run directly with `WRITE_ACTUAL` and verify with `git diff` before `git add`.